src/components/pages/RoleApply/Applications/Mentee.js

```diff
@@ -24,7 +24,8 @@ import {
 } from '@ant-design/icons';
 
 import { states, countries, tech_stack } from '../../../common/constants';
-import './Styles/menteeApplication.css';
+import './Styles/mentorApplication.css';
```

Comment:
Great job implementing AntDesign and style for this page. also thanks for cleaning up the code rather than commenting out. Clean and concise.